docs/configuration/http_filters/ip_tagging_filter.rst

```diff
@@ -0,0 +1,45 @@
+.. _config_http_filters_ip_tagging:
+
+Ip tagging filter
+====================
+
+This is a filter which enables Envoy to tag requests with extra information such as location, cloud source and any
+extra data. This is useful to prevent against DDoS.
+
+**Note**: this filter is under active development, and currently does not perform any tagging on requests. In other
+words, installing this filter is a nop in the filter chain.
+
+.. code-block:: json
+
+  {
+    "type": "decoder",
+    "name": "ip_tagging",
+    "config": {
+      "request_type": "...",
+      "ip_tags": []
+    }
+  }
+
+request_type:
+  (optional, string) The type of requests the filter should apply to. ``external``, ``internal``, or ``both``.
```

Comment:
Can you expand the about what is deemed internal/external? ie take it from the rl filter.